aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -2995,7 +2995,7 @@ void quantize_tensor_per_tensor_affine_sub_byte_cpu(
       const auto elem_per_byte = CHAR_BIT / bit_width;
       for (int i = 0; i < numel; ++i) {
         float inv_scale = scale == 0 ? 1.0f : 1.0f / scale;
-        int qvalue = lrintf(std::nearbyint(rdata[i] * inv_scale) + zero_point);
+        int64_t qvalue = lrintf(std::nearbyint(rdata[i] * inv_scale) + zero_point);
```

Comment:
Why this change?

From a style perspective this is why I prefer auto.

Reply:
Same as https://github.com/pytorch/pytorch/pull/51792#discussion_r607901624

Reply:
This is already done in https://github.com/pytorch/pytorch/pull/55638